src/core_plugins/kibana/public/dashboard/partials/save_dashboard.html

```diff
@@ -11,12 +11,18 @@
     placeholder="Dashboard title"
     input-focus="select"
   >
+
+  <saved-object-save-as-check-box saved-object="opts.dashboard"></saved-object-save-as-check-box>
```

Comment:
I'm seeing a slight flicker when saving a Dashboard (or other object) for the first time. You can see the "Save as a new dashboard" checkbox flicker for a moment before the dropdown is closed:

![save_as_flicker](https://cloud.githubusercontent.com/assets/1238659/20576888/a57ad92a-b174-11e6-9d00-a9dee571d8e9.gif)

I think we just need to add a "isSaving" flag to the inherited scope. It will default to false, we can set it to true while the save request is being sent, and then back to false once the response is received.

Then we can pass it to the `savedObjectSaveAsCheckBox` directive:

```html
<saved-object-save-as-check-box
  saved-object="opts.dashboard"
  is-saving="opts.isSaving"
></saved-object-save-as-check-box>
```

The directive can then hide itself when `isSave` is true:

```html
<div ng-hide="!savedObject.id || isSaving">
  <!-- ... -->
</div>
```